<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateManualEulerRingFormClass</class>
 <widget class="QDialog" name="CreateManualEulerRingFormClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CreateManualEulerRingForm</string>
  </property>
  <widget class="QFrame" name="frameLibFolder">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibName">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Library Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="EditLibID">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>12345</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>名称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetLibList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>191</width>
     <height>181</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Name</string>
    </property>
   </column>
  </widget>
  <widget class="QFrame" name="frame_20">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>371</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_74">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>141</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>輝
度</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="framePanel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_67">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>輝度マージン</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditMergin">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>膨張ドット数</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditExpandedDot">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>99999999</number>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_25">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="label_90">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>20</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>補
正</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="indent">
     <number>1</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>371</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="label_91">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>20</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>輝
度</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="indent">
      <number>1</number>
     </property>
    </widget>
   </widget>
   <widget class="QSpinBox" name="EditAdjustBlack">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="EditAdjustWhite">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_92">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>明側　最大補正輝度量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_93">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>暗側　最大補正輝度量</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>キャンセル</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyEulerRing.qrc">
     <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>360</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>作成</string>
   </property>
   <property name="icon">
    <iconset resource="PropertyEulerRing.qrc">
     <normaloff>:/Qtres/Make.png</normaloff>:/Qtres/Make.png</iconset>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="PropertyEulerRing.qrc"/>
 </resources>
 <connections/>
</ui>
